roles/update/tasks/l3_agent_connectivity_check_stop.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: stop l3 agent connectivity check
```

Comment:
I see that there is task for gathering log for instance creation, but we don't have gathering log after ping test end.

Reply:
The logs of the ping are in ci-framework-data/tests/updates/ping*.log and they are saved automatically.